public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx

```diff
@@ -35,8 +35,10 @@ export function AdvancedOptions({ query, onChange }: Props) {
       >
         <div>
           <InlineFieldRow>
-            <InlineField label="Min Duration" labelWidth={21} grow>
+            <InlineField htmlFor="minDuration" label="Min Duration" labelWidth={21} grow>
               <Input
+                id="minDuration"
+                name="minDuration"
```

Comment:
Just supplying an `id` for `Input` should be enough I think, as `InlineField` looks at its children's props.

Reply:
Gotcha, I didn't realize InlineField did that, that's pretty cool. I'll update these